support-frontend/app/controllers/Promotions.scala

```diff
@@ -30,15 +30,15 @@ class Promotions(
 
   implicit val a: AssetsResolver = assets
 
-  def promo(promoCode: String): Action[AnyContent] = CachedAction() { implicit request =>
+  def promo(promoCode: String, orderIsAGift: Boolean = false): Action[AnyContent] = CachedAction() { implicit request =>
```

Comment:
I'm not sure how this parameter would ever be used, I think we would probably want to work out whether a promo applies to a gift or not based on the plan it's set up for. It might be worth removing this for now and just passing false into the redirect

Reply:
Yes! I wanted to talk to you about this as I wasn't sure whether promotions will be applied to gifts. Do you think it's better to remove it?

Reply:
I think so yes, and then we can work it all out properly when we look at promos for gifting in future